info.toml

```diff
@@ -221,6 +221,20 @@ in the book's section References and Borrowing':
 https://doc.rust-lang.org/book/ch04-02-references-and-borrowing.html#mutable-references.
 """
 
+[[exercises]]
+name = "move_semantics6"
+path = "exercises/move_semantics/move_semantics6.rs"
+mode = "compile"
+hint = """
+Here if you want to find the answer by yourself, you can read in the book's section References and Borrowing:
+https://doc.rust-lang.org/stable/book/ch04-02-references-and-borrowing.html
+Else the first problem is that `get_char` is taking ownership of the string
+So `data` is moved and can't be used for `string_uppercase`
```

Comment:
I would rewrite everything after this to not explicitly tell the solution to the user, e.g.:

`data` is moved to `get_char` first, meaning that `string_uppercase` cannot manipulate the data. Once you've fixed that, `string_uppercase`'s function signature will also need to be adjusted - can you figure out how? Another hint: it has to do with the `&` character.

Reply:
At first, I was looking to do something like that, but I found it very frustrating that you didn't have a solution to the exercise with the best methods to solve it. So I gave it in the hints.

Maybe it could be a feature.
Because with the correct answer, you could see what you did wrong or could upgrade, and it could work like that? 
`rustlings solve <exercise>` and it's print the answer.